XRTK-Core/Packages/manifest.json

```diff
@@ -15,8 +15,6 @@
     "com.unity.ide.vscode": "1.2.3",
     "com.unity.mobile.android-logcat": "1.2.1",
     "com.unity.test-framework": "1.1.24",
-    "com.unity.textmeshpro": "2.1.4",
-    "com.unity.ugui": "1.0.0",
```

Comment:
Does this not cause issue, removing the Unity UI system?  Granted Core doesn't use it

Reply:
No bc the dependency is defined in the project.json

Reply:
Just means one less place to have to update the information